README.md

```diff
@@ -1 +1,63 @@
 # CODE Library backend
+
+This repository contains the backend components of the library system at the CODE University of 
+Applied Sciences, Berlin. The aim of this project is, to provide a library system for the management 
+and borrowing of books. This includes notifications to users about available and overdue books.
+
+## System Architecture
+
+The backend consists of five microservices. Each microservice resides in its own sub-directory.
+The directories are named accordingly to the name of the microservice.
+Please refer to the table below, to get more details.
+
+| Service | Description |
+| --- | --- |
+| identity | Implements the authentication and authorization functionalities |
+| book | Implements the book management functionalities |
+| borrow | Implements the borrow management functionalities |
+| notification | Implements the user notification functionalities |
+| api | Implements the REST API for user access |
+
+The services communicate through remote-procedure-calls (RPC) among each other. However, only the
```

Comment:
nitpick: Maybe we should mention that this feature is not currently implemented? Otherwise people will spend time looking for implementation only to realize it's not there yet :)
```suggestion
The services communicate through remote-procedure-calls (RPC)(**it's currently WIP**) among each other. However, only the
```

Reply:
I removed this implementation specific part and mentioned it in the project description. Also changed a few other parts.

Reply:
:+1: